plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoIcebergRestCatalogFactory.java

```diff
@@ -70,6 +71,7 @@ public TrinoIcebergRestCatalogFactory(
         this.warehouse = restConfig.getWarehouse();
         this.sessionType = restConfig.getSessionType();
         this.vendedCredentialsEnabled = restConfig.isVendedCredentialsEnabled();
+        this.prefix = restConfig.getPrefix();
```

Comment:
nit: Move to under `this.warehouse = ...` for consistency with field orders.

Reply:
done